build.gradle

```diff
@@ -37,7 +37,7 @@ plugins {
 buildDir = 'gradle-build'
 
 ext {
-    opensearchVersion = '1.0.0.0-beta1'
+    opensearchVersion = '1.0.0.0-rc1'
```

Comment:
Looks like this is opensearch-sercurity version and not opensearch version. Can you rename the variable to avoid confusion ?

Reply:
Changed.